src/applications/facility-locator/components/LocationHours.jsx

```diff
@@ -1,107 +1,76 @@
-import { values, every, capitalize } from 'lodash';
-import React, { Component } from 'react';
-import * as Sentry from '@sentry/browser';
-import moment from 'moment';
+import { get } from 'lodash';
+import React from 'react';
 
 /**
  * VA Facility Known Operational Hours
  */
-export default class LocationHours extends Component {
-  formatTimeRange(hour) {
-    const hours = hour.split('-').map(time => moment(time, 'hmmA'));
-    const isValid = hours.every(time => time.isValid());
-
-    if (!isValid) {
-      Sentry.withScope(scope => {
-        scope.setExtra('data', this.props.location);
-        Sentry.captureMessage('API location hours data is malformed');
-      });
-
-      return '';
-    }
-
-    return hours.map(time => time.format('h:mmA')).join(' - ');
-  }
-
-  formatLocationHours(hours) {
-    return Object.keys(hours).reduce((accum, key) => {
-      if (hours[key] === '-' || hours[key] === 'Closed') {
-        return { ...accum, [key]: 'Closed' };
-      }
+const LocationHours = ({ location }) => {
+  // Derive the hours for each day.
+  const hoursInfo = get(location, 'attributes.hours');
+  const sunday = get(hoursInfo, 'sunday', 'N/A');
+  const monday = get(hoursInfo, 'monday', 'N/A');
+  const tuesday = get(hoursInfo, 'tuesday', 'N/A');
+  const wednesday = get(hoursInfo, 'wednesday', 'N/A');
+  const thursday = get(hoursInfo, 'thursday', 'N/A');
+  const friday = get(hoursInfo, 'friday', 'N/A');
+  const saturday = get(hoursInfo, 'saturday', 'N/A');
+
+  return (
+    <div>
+      <h4 className="highlight">Hours of Operation</h4>
+
```

Comment:
Do we need to add a condition for when `hoursInfo` is falsy?

Reply:
That's a great question. Currently if `hoursInfo` is undefined, we show something like:

```
Sunday: N/A
Monday: N/A
Tuesday: N/A
Wednesday: N/A
...
```

Should we show nothing instead? I think this might be more of a design-related question -- do you perhaps know someone we can tag to get their input on this? :)